crates/extensions/tedge_http_ext/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+mod actor;
+mod messages;
+
+pub use messages::*;
+
+use actor::*;
+use tedge_actors::{new_mailbox, Address, Mailbox, Recipient, RuntimeError, RuntimeHandle};
+
+pub struct HttpActorInstance {
+    actor: HttpActor,
+    mailbox: Mailbox<(usize, HttpRequest)>,
+    address: Address<(usize, HttpRequest)>,
+    clients: Vec<Recipient<Result<HttpResponse, HttpError>>>,
+}
+
+impl HttpActorInstance {
+    pub fn new(config: HttpConfig) -> Result<Self, HttpError> {
+        let actor = HttpActor::new(config)?;
+        let (mailbox, address) = new_mailbox(10);
+
+        Ok(HttpActorInstance {
+            actor,
+            mailbox,
+            address,
+            clients: vec![],
+        })
+    }
+
+    pub fn add_client(
```

Comment:
Eager to see the usage of this API from a consumer Actor. That probably will clarify why this API is designed like this to accept the `Recipient` as a parameter **from** the consumer rather than creating and returning one **to** the consumer.

Reply:
True, this must be done asap.